tests/lean/run/let_coe_sort.lean

```diff
@@ -0,0 +1,24 @@
+inductive foo | bar
+
+instance : has_coe_to_sort foo :=
+⟨Type, λ _, unit⟩
+
+set_option pp.all true
+
+example : true :=
+begin
+  let x : foo.bar := (),
+  have y : foo.bar := (),
```

Comment:
Is it worth adding a test for `suffices` too?

Reply:
Feel free to add one.